src/lib/components/notifications.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import { notifications } from '$lib/stores/notifications';
+  import { crossfade } from 'svelte/transition';
+  import { quintOut } from 'svelte/easing';
+  import { flip } from 'svelte/animate';
+
+  const [send, receive] = crossfade({
+    duration: (d) => Math.sqrt(d * 100),
+
+    fallback(node) {
+      const style = getComputedStyle(node);
+      const transform = style.transform === 'none' ? '' : style.transform;
+
+      return {
+        duration: 600,
+        easing: quintOut,
+        css: (t) => `
+					transform: ${transform} scale(${t});
```

Comment:
nit: curious what the arguments for tabs vs spaces would be for cases when the code/markup is plain text